framework/src/st_things/things_stack/framework/things_data_manager.c

```diff
@@ -1928,10 +1931,10 @@ int dm_load_legacy_cloud_data(es_cloud_signup_s **cl_data)
 	return ret;
 }
 
-const char *dm_get_things_device_type(int device_id)
+const char *dm_get_things_device_type(int *device_id)
```

Comment:
Use 'char *'

Reply:
This function was modify in other PR. I will remove '*'